nbconvert/_version_tools.py

```diff
@@ -0,0 +1,83 @@
+# import pep440 # cannot be imported as described below
+import re 
```

Comment:
You may also just copy past the `is_cannonical` from [pep 440 itself](https://www.python.org/dev/peps/pep-0440/#appendix-b-parsing-version-strings-with-regular-expressions) that was added recently to make this kind of things easier. 

I'll make my pep440 easier to vendor.